src/http/cookie.cr

```diff
@@ -53,9 +58,20 @@ module HTTP
       "#{@name}=#{URI.escape value}"
     end
 
-    def expired?
-      if e = expires
-        e < Time.utc
+    def expiration_time(time_reference = @creation_time)
+      if max_age = @max_age
+        time_reference + max_age
+      else
+        @expires
+      end
+    end
+
+    # Returns the expiration status of this cookie as a `Bool` given a creation time
+    def expired?(time_reference = @creation_time)
+      if @max_age == 0.seconds
+        true
+      elsif (time = expiration_time(time_reference)) && time < Time.now
```

Comment:
Maybe use `Time.utc_now` to avoid fetching location information? Not sure.

Reply:
Good question..

Reply:
Yes, please use `Time.utc` everywhere.